On v3.2.4, when using monad wrappers, haskell errors with a variable not in scope (`alexEOF`).

Last week I was using the `monadUserState` wrapper and I found the following error when compiling the .hs file:
```text
/path/to/my/project/templates/wrappers.hs:243:16: error:
    • Variable not in scope: alexEOF :: Alex AlexUserState
    • Perhaps you meant data constructor ‘AlexEOF’ (line 2015)
```
then I looked at the generated .hs file and I see that, in effect, the variable was not defined.

[This](https://stackoverflow.com/questions/28141182/what-do-i-need-to-add-to-use-monaduserstate-with-alex-when-parsing) stackoverflow answer states that the `alexEOF` should be defined, but that doesn't appear on the alex documentation.

--- a/doc/alex.xml
+++ b/doc/alex.xml
@@ -1408,6 +1408,13 @@ alexError        :: String -> Alex a
 alexGetStartCode :: Alex Int
 alexSetStartCode :: Int -> Alex ()</programlisting>
 
+	<para>The <literal>monad</literal> wrapper expects that you
+	define a variable <literal>alexEOF</literal> with the following
+	signature:</para>
+
+<programlisting>alexEOF :: Alex result</programlisting>
+
+
 	<para>To invoke a scanner under the <literal>monad</literal>
 	wrapper, use <literal>alexMonadScan</literal>:</para>
 

